app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please add `:confirmable` to confirm the confirmation email also this is a requirement so kindly add it :slightly_smiling_face: 

![Screenshot from 2023-11-18 15-31-49](https://github.com/sheezu900/blog-rails/assets/45226848/c4b1b2f3-df1d-469c-8fda-72c7d1b541b8)